inspirehep/modules/workflows/views/callback.py

```diff
@@ -97,7 +148,13 @@ def webcoll_callback():
 
     Expects the request data to contain a list of record ids in the
     recids field.
+
+    DEPRECATED: replaced by the simpler continue_workflow_callback
```

Comment:
Well the code was not there at the beginning, then people were confused so we had to add it.

I think it's a basic usability question here. If one receives an email saying that the record is ready and a link to it, it's expected to be there (no need to overcomplicate it saying it will be there eventually). Since it is working nowadays, I don't see a reason to go backwards.

And we know by past experience that 'it runs every 5 minutes' it's quite an illusion. There are many times things happening that slow down the task queue so this time frame is longer.